pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[tool.poetry]
+name = "vc-authn-oidc"
+version = "0.2.2"
+description = "Verifiable Credential Identity Provider for OpenID Connect."
+authors = ["TODO"]
```

Comment:
Should probably put the `Government of British Columbia <[DItrust@gov.bc.ca](mailto:DItrust@gov.bc.ca)>` here.